### ISSUE TITLE
Export problem of images

### ISSUE THREAD

**Issue by zabih036** [BALKANGraph/OrgChartJS#839]
export of chart with images not showing in export file

**Comment by ZornitsaPesheva** [comment]
To be able to export images, they need to be public, i.e. you should be able to browse these images in any browser.
Here you can read about these requirements: https://balkan.app/OrgChartJS/Docs/Exporting#server

**Comment by zabih036** [comment]
Thanks so much for the quick response.
I tried it work in svg but not in png format and pdf format
so thanks again
Regards

**Comment by ZornitsaPesheva** [comment]
Yes, that's because only the PDF and PNG exports are done on the export server. The other exports are done in the browser.

**Comment by zabih036** [comment]
Ok, thanks for the clarification.
